README.md

```diff
@@ -14,9 +14,51 @@ Within vim, press Space and Backspace (in normal mode) to move forwards and back
 
 Especially useful for live demos with a lot of canned code, which you want to present and run incrementally.
 
+You only need one file. That file will be your slides (notice the plural). Each
+change and save will create a new commit. Moving back and forth between slides
+will checkout the current working directory to the previous commit and next
+commit respectively and reload the buffer in vim.
 
 Installation
 ------------
 
-Copy `plugin/git-slides.vim` to your `~/.vim/plugin`
-folder, and put `bin/git-slides` somewhere on your path.
+1. Copy `plugin/git-slides.vim` to your `~/.vim/plugin` or use a plugin manager. For example with [vim-plug](https://github.com/junegunn/vim-plug) add the following line to your vimrc (assuming you already have vim-plug installed):
+```vimscript
+Plug 'gelisam/git-slides'
+```
+2. Put [git-slides.sh](bin/git-slides) in your path. For example if you installed with `vim-plug`:
```

Comment:
you're calling it `git-slides.sh` but pointing to a file named `git-slides`, is that a typo?

Reply:
Typo. Fixed